appengine/Dockerfile

```diff
@@ -54,7 +55,7 @@ ENV PATH /rbenv/shims:/rbenv/bin:$PATH
 
 # Preinstall ruby runtimes.
 # The LAST version in the list is set as the default.
-ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p647 2.1.7 2.2.3
+ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p648 2.1.8 2.2.3 2.2.4
```

Comment:
Let me know before you fire off the jenkins job to push the image.  I'll send an announcement out to the slack/gg with the version changes.  Maybe we should use the release feature in GitHub for this as well?